packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1613,6 +1613,17 @@ export default class BackupNg {
                     _ => _.mode === 'delta' && _.scheduleId === scheduleId
                   )
                 ): any)
+
+                // FIXME: implement optimized multiple VHDs merging with
+                // synthetic delta
+                //
+                // For the time being, limit the number of deleted backups to 2
+                // because it can time a very long time and can lead to
+                // interupted backup with broken VHD chain.
+                if (oldBackups.length > 2) {
+                  oldBackups.length = 2
```

Comment:
[Here](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xo-mixins/backups-ng/index.js#L1303), We will delete all `oldBackups`, no?

Reply:
The code you linked is for full backups, not delta backups, which are the only ones concerned by this change.

Reply:
I wanted to link [this one](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xo-mixins/backups-ng/index.js#L1626). 
It's my fault ^_^

Reply:
Exactly, `deleteOldBackups` will remove all backups in the `oldBackups` list.

In this PR, I'm resizing this list to make sure that there is, at most, two entries in it.